Makefile

```diff
@@ -50,26 +55,29 @@ ecr-credential-provider.exe: $(wildcard ./cmd/ecr-credential-provider/*.go)
 
 .PHONY: docker-build-amd64
 docker-build-amd64:
-	docker buildx build --output=type=docker \
+	docker buildx build \
+		--load \
 		--build-arg VERSION=$(VERSION) \
 		--build-arg GOPROXY=$(GOPROXY) \
 		--platform linux/amd64 \
 		--tag $(IMAGE) .
 
 .PHONY: docker-build-arm64
 docker-build-arm64:
-	docker buildx build --output=type=docker \
+	docker buildx build \
+		--load \
 		--build-arg VERSION=$(VERSION) \
 		--build-arg GOPROXY=$(GOPROXY) \
 		--platform linux/arm64 \
 		--tag $(IMAGE) .
 
-.PHONY: docker-build
-docker-build:
-	docker buildx build --output=type=registry \
-		--build-arg LDFLAGS=$(LDFLAGS) \
+.PHONY: docker-build-push
+docker-build-push:
+	docker buildx build \
+		--output=type=registry \
+		--build-arg VERSION=$(VERSION) \
```

Comment:
I am wondering a bit about using `VERSION` here. That means that the staging versions will keep overriding the _last_ tag that was made in git. I would rather use what would be the _next_ tag. Either that or create a patch suffix.